charts/base-cluster/templates/tetragon/tetragon.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.tetragon.enabled }}
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: tetragon
+  namespace: kube-system
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: tetragon
+spec:
+  chart:
+    spec: {{- include "base-cluster.helm.chartSpec" (dict "repo" "tetragon" "chart" "tetragon" "context" $) | nindent 6 }}
+  interval: 5m
+  driftDetection:
+    mode: enabled
+  {{- if .Values.monitoring.prometheus.enabled }}
+  dependsOn:
+    - name: kube-prometheus-stack
+      namespace: monitoring
+  {{- end }}
+  values:
+    export:
+      stdout:
+        image:
+          repository: {{ .Values.global.imageRegistry }}
```

Comment:
Why did you change this from `repository: {{ printf "%s/cilium/hubble-export-stdout" ($.Values.global.imageRegistry | default (include "base-cluster.defaultRegistry" (dict))) }}`?

Reply:
changed it to an `if` statement